.github/workflows/test.yml

```diff
@@ -69,7 +69,7 @@ jobs:
               - ".cargo/**"
               - 'Cargo.toml'
               - 'Cargo.lock'
-              - 'rust-toolchain'
+              - 'rust-toolchain.toml'
```

Comment:
Merged in https://github.com/vectordotdev/vector/pull/11974 to fix `make check-clippy` and builds.